Assets/Sprites/Background/enlarged_sprites/grass.png.meta

```diff
@@ -45,11 +45,11 @@ TextureImporter:
   compressionQuality: 50
   spriteMode: 1
   spriteExtrude: 1
-  spriteMeshType: 1
+  spriteMeshType: 0
```

Comment:
FYI if the sprite has a tight mesh it reduces overdraw (and improves performance) vs Full Rect as Unity can avoid rendering the grass in areas where it's fully transparent. This might not be great, but I'm not sure 9-slicing would work with it.

Could alternatively split this into two sprites.